### ISSUE TITLE
Custom CHAR_SET using list mode is not getting the right subarray of characters provided.

### ISSUE THREAD

**Issue by juancwu** [zero-to-mastery/ascii-art-2022#5]
The way the current code is set up to be is splitting the giving CHAR_SET in the format `[a,b,c,d]` using a coma will yield a list of string as follow => `['[a','b','c','d]']`. Then the code is getting a sub-array using `list(CHAR_SET)[1:-1]` will yield `['b', 'c']`. As you can see `a` and `d` are missing from the list.

![image](https://user-images.githubusercontent.com/46619361/193432166-d11805c9-1183-4cdb-87ec-e4008ede95d9.png)

Solution:
Before splitting the string we should get a sub-string with only comma separating the characters.
```python
CHAR_SET = "[a,b,c,d]"
CHAR_SET = CHAR_SET[1:-1] # this will be "a,b,c,d"

# now we can just split the string
CHAR_SET = CHAR_SET.split(",") # this will be ['a','b','c','d']
```
![image](https://user-images.githubusercontent.com/46619361/193432253-c1cd7dc0-0042-45d2-ae36-3de3e00fc312.png)

**Comment by beneyal** [comment]
Nice catch! Want to open a PR with the fix? 🙂 Just make sure to pull the latest changes, I just merged your current PR. Thanks! 🙏

**Comment by juancwu** [comment]
Sure, I can do that.

**Comment by juancwu** [comment]
A PR has been made for this issue #7 .

**Comment by beneyal** [comment]
Closed by #7 🙂
